ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/resources/event.clj

```diff
@@ -99,7 +100,21 @@
 ;;
 ;; collection
 ;;
+
+(defn compare-timestamps-desc
+  [e1 e2]
+  (compare (:timestamp e2) (:timestamp e1)))
+
+(defn sort-by-timestamps
+  [events]
+  (sort compare-timestamps-desc events))
+
+(defmethod crud/sort-collection resource-name
+  [request collection]
+  (sort-by-timestamps collection))
```

Comment:
Couldn't this line be written as:

``` clojure
(sort-by :timestamp coll)
```

so that both helper functions `compare-timestamps-desc` and `sort-by-timestamps` aren't needed?